### DISCUSSION TITLE
Bug request

### DISCUSSION THREAD

**Discussion by hadooooken** [PlaceholderAPI/PlaceholderAPI#694] [(2) Support] [answered]
Hello developers! I found a bug in the plugin that prevents the use of third-party extensions.
![jWgjPDIsrdI](https://user-images.githubusercontent.com/70557914/129531032-afa19d20-9a6e-477f-bd09-32417116638a.jpeg)

**Comment by Andre601** [comment, marked as answer]
Please use the issue tracker for bug reports.
Also, this does not look like a PlaceholderAPI issue at all.
